test/e2e/scalability/density.go

```diff
@@ -777,9 +798,9 @@ var _ = SIGDescribe("Density", func() {
 					framework.ExpectNoError(framework.DeleteRCAndWaitForGC(c, rcNameToNsMap[name], name))
 				}
 				workqueue.Parallelize(25, nodeCount, deleteRC)
+				latencyMeasurementPhase.End()
 			}
-
-			cleanupDensityTest(dConfig)
+			cleanupDensityTest(dConfig, testPhaseDurations.StartPhase(900, "cleanup"))
```

Comment:
I would suggest passing `testPhaseDurations` instead of just a phase. As we might want to create more than one phases inside that function (as I suggest in another comment).

Reply:
Also could you make the msg better and probably split further into something like:
- saturation pods deletion
- configmaps, secrets and daemons deletion

Reply:
Done.